### DISCUSSION TITLE
Request for Arm64 Images

### DISCUSSION THREAD

**Discussion by ximatar** [accetto/ubuntu-vnc-xfce-g3#17] [Ideas]
Hi,

Can you please also create images for arm64?  I would very much appreciate this.

Regards.

**Comment by accetto** [comment]
Hello @ximatar ,

sorry to be so late, but I haven't noticed you entry. :(

Unfortunatelly, I've already tried to build images for Arm64, but I've failed. So I've decided to leave it off for now. You can check the issue #8 for more info.

I really appreciate your feedback. Thank you.

Regards,
accetto
